src/query/sql/src/planner/binder/bind_query/bind_value.rs

```diff
@@ -402,6 +402,20 @@ pub fn bind_values(
     for (row_idx, row) in values.iter().enumerate() {
         for (column_idx, expr) in row.iter().enumerate() {
             let (scalar, data_type) = scalar_binder.bind(expr)?;
+            if matches!(
+                scalar,
+                ScalarExpr::WindowFunction(_)
+                    | ScalarExpr::AggregateFunction(_)
+                    | ScalarExpr::SubqueryExpr(_)
+                    | ScalarExpr::UDAFCall(_)
+                    | ScalarExpr::UDFCall(_)
```

Comment:
better to use `ScalarExpr` `evaluable` function to check, set returning function and async function are not allowed too.